.circleci/config.yml

```diff
@@ -0,0 +1,429 @@
+          - DATABASE_URL=postgres://postgres@localhost/statesman_test
+      - image: circleci/postgres:9.6
+        environment:
+          - POSTGRES_USER=postgres
+          - POSTGRES_DB=statesman_test
+      - image: circleci/mongo:3.0.14
+      - image: circleci/mysql:5.7.18
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=true
+          - MYSQL_USER=root
+          - MYSQL_PASSWORD=
+          - MYSQL_DATABASE=statesman_test
+    steps: *steps
+  build-ruby227-rails-505-mysql:
+    docker:
+      - image: circleci/ruby:2.2.7-node
+        environment:
+          - RAILS_VERSION=5.0.5
+          - DATABASE_URL=mysql2://root@127.0.0.1/statesman_test
+          - EXCLUDE_MONGOID=true
+      - image: circleci/postgres:9.6
+        environment:
+          - POSTGRES_USER=postgres
+          - POSTGRES_DB=statesman_test
+      - image: circleci/mysql:5.7.18
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=true
+          - MYSQL_USER=root
+          - MYSQL_PASSWORD=
+          - MYSQL_DATABASE=statesman_test
+    steps: *steps
+  build-ruby227-rails-505-postgres:
+    docker:
+      - image: circleci/ruby:2.2.7-node
+        environment:
+          - RAILS_VERSION=5.0.5
+          - DATABASE_URL=postgres://postgres@localhost/statesman_test
+          - EXCLUDE_MONGOID=true
+      - image: circleci/postgres:9.6
+        environment:
+          - POSTGRES_USER=postgres
+          - POSTGRES_DB=statesman_test
+      - image: circleci/mysql:5.7.18
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=true
+          - MYSQL_USER=root
+          - MYSQL_PASSWORD=
+          - MYSQL_DATABASE=statesman_test
+    steps: *steps
+  build-ruby227-rails-513-mysql:
+    docker:
+      - image: circleci/ruby:2.2.7-node
+        environment:
+          - RAILS_VERSION=5.1.3
+          - DATABASE_URL=mysql2://root@127.0.0.1/statesman_test
+          - EXCLUDE_MONGOID=true
+      - image: circleci/postgres:9.6
+        environment:
+          - POSTGRES_USER=postgres
+          - POSTGRES_DB=statesman_test
+      - image: circleci/mysql:5.7.18
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=true
+          - MYSQL_USER=root
+          - MYSQL_PASSWORD=
+          - MYSQL_DATABASE=statesman_test
+    steps: *steps
+  build-ruby227-rails-513-postgres:
+    docker:
+      - image: circleci/ruby:2.2.7-node
+        environment:
+          - RAILS_VERSION=5.1.3
+          - DATABASE_URL=postgres://postgres@localhost/statesman_test
+          - EXCLUDE_MONGOID=true
+      - image: circleci/postgres:9.6
+        environment:
+          - POSTGRES_USER=postgres
+          - POSTGRES_DB=statesman_test
+      - image: circleci/mysql:5.7.18
+        environment:
+          - MYSQL_ALLOW_EMPTY_PASSWORD=true
+          - MYSQL_USER=root
+          - MYSQL_PASSWORD=
+          - MYSQL_DATABASE=statesman_test
+    steps: *steps
+
+workflows:
+  version: 2
+  tests:
+    jobs:
+      - build-ruby241-rails-429
```

Comment:
How come we have a standalone build for rails `4.2.9` but no equiavalent for `5.0.5` or `5.1.3`?

Reply:
We came to the conclusion that the purpose of this build is to test the Mongo adapter. The rest of the tests run in every case, so there is a fair bit of duplication.

An interesting future refactor would be to split up the tests a bit more logically so you can have a standalone set of tests and then tests for each adapter. This would be quite a bit of work, but would make more sense.

I'm converted the standalone build to explicitly be a Mongo build, and have added a Mongo build for Rails 5.0.x and 5.1.x, which were conspicuously absent.